client/src/components/SitterSignup/SitterSignup.tsx

```diff
@@ -0,0 +1,63 @@
+import { Box, Switch, Typography, Grid, Divider, Button } from '@material-ui/core';
+import { useState, useEffect } from 'react';
+import { useHistory, Link } from 'react-router-dom';
+import { useAuth } from '../../context/useAuthContext';
+import axios from 'axios';
+
+export default function SitterSignup(): JSX.Element {
+  const history = useHistory();
+
+  const [checked, setChecked] = useState(false);
+
+  useEffect(() => {
+    async function getUser() {
+      await axios.get('/users/isSitter/').then((response: any) => {
+        if (response.data.isSitter) {
+          setChecked(response.data.isSitter);
+        }
+      });
+    }
+    getUser();
+  }, [history]);
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const checked = event.target.checked;
+
+    async function updateIsSitter() {
+      await axios.patch('/users/isSitter/', { isSitter: checked });
+    }
+    updateIsSitter();
+    setChecked(checked);
+  };
+
+  return (
+    <Grid container justify="center">
+      <Grid item xs={12}>
+        <Typography variant={'h5'}>
+          Pet sitters play an important role in our community. Pets need friends to love while their owners are away!{' '}
+        </Typography>
+        <Typography variant={'h6'}>Toggle this to become a pet sitter today!</Typography>
+      </Grid>
+      <Divider />
+      <Grid item xs={12}>
+        <Typography>
+          <Switch onChange={handleChange} checked={checked} />
+          Pet sitter toggle
+        </Typography>
+      </Grid>
+      <Grid item xs={12}>
+        {checked == true && (
+          <Box>
+            <Typography variant={'h6'}>
+              You are NOW signed up as a pet Sitter. Please fill out your profile so you may be contacted by people
```

Comment:
Also a heads up, you can wrap this up in an element/component to capitalize it through CSS - this would be preferable from an accessibility perspective.

Reply:
I'll take a look in to that if I come across it in a future feature, thanks!